src/sentry/utils/snuba.py

```diff
@@ -94,6 +94,49 @@ class SnubaError(Exception):
     pass
 
 
+class UnqualifiedQueryError(SnubaError):
+    """
+    Exception raised when no project_id qualifications were provided in the
+    query or could be derived from other filter criteria.
+    """
+    pass
```

Comment:
nit: You're a bit inconsistent with `pass` usage. Obviously doesn't matter.

Reply:
Oops, will fix

Reply:
Fixed in fa5db89, thanks